lib/kino_explorer/data_transform_cell.ex

```diff
@@ -643,4 +656,11 @@ defmodule KinoExplorer.DataTransformCell do
   defp normalize_values_from(values) when is_list(values), do: values
   defp normalize_values_from(nil), do: []
   defp normalize_values_from(values), do: [values]
+
+  defp get_distinct(df) do
+    for {col, type} <- DataFrame.dtypes(df), type == :string, into: %{} do
+      values = DataFrame.distinct(df, [col]) |> DataFrame.to_columns() |> Map.get(col)
+      {col, values}
+    end
```

Comment:
We are computing the distinct for the whole series but we only use a single column, we likely want to do this instead:

```elixir
try do
  df[col] |> Series.distinct() |> Series.series() |> Enum.to_list()
rescue
  _ -> []
end
```

I added a try/catch because we don't want to raise if the dataframe is lazy (please add a test). :)